server/pbench/bin/pbench-report-status

```diff
@@ -21,8 +21,12 @@ while true; do
             shift 2
             ;;
         -t|--timestamp)
+            # timestamp is of the form run-<yyyy>-<mm>-<dd>T<HH>:<MM>:<SS>-<TZ>
+            # It's the variable TS created by pbench-base.sh.
+            # Snip off the initial "run-"
             timestamp="${2#run-}"
-            timestamp="${timestamp%-UTC}"
+            # Snip off the trailing "-<TZ>" - assumes that <TZ> does not contain a "-"
+            timestamp="${timestamp%-*}"
```

Comment:
Are we sure we want to do this here?  It looks like `$timestamp` is used as a parameter to `pbench-gen-json-payload` where we want to make sure the payload's timestamp is in UTC.

So can we consider moving this stripping to line 89?

Reply:
OK